assets/src/components/v2/elevator/elevator_closures.tsx

```diff
@@ -177,10 +196,92 @@ const OutsideClosureList = ({
           <div className="more-elevators-text">
             +{numOffsetRows} more elevators
           </div>
-          <div className="paging-indicators">
-            {getPagingIndicators(numPages)}
+          <PagingIndicators numPages={numPages} pageIndex={pageIndex} />
+        </div>
+      )}
+    </div>
+  );
+};
+
+interface OutsideClosuresViewProps extends OutsideClosureListProps {}
+
+const OutsideClosuresView = ({
```

Comment:
Same comment as with the stylesheets — these "views" seem so separate, and each complex enough, that they deserve their own modules. (This also makes me wonder about making them separate widgets on the backend, if that would make sense.)